.eslintrc.js

```diff
@@ -0,0 +1,70 @@
+module.exports = {
```

Comment:
I don't think this file should be checked in. In the least it should be added to gitignore, but how did it get generated in the first place?